cellpack/bin/simularium_converter.py

```diff
@@ -37,7 +39,7 @@ class ConvertToSimularium(argparse.Namespace):
     DEFAULT_OUTPUT_DIRECTORY = "/Users/meganriel-mehan/Dropbox/cellPack/"
     DEFAULT_INPUT_RECIPE = "/Users/meganriel-mehan/dev/allen-inst/cellPack/cellpack/cellpack/test-recipes/NM_Analysis_FigureC1.json"
     DEFAULT_GEO_TYPE = "OBJ"  # Other options: SPHERE or PDB
-    DEFAULT_SCALE_FACTOR = 1.0 / 10
+    DEFAULT_SCALE_FACTOR = 1.0 # / 10
```

Comment:
we need this scaling for pdbs. Was it causing any problems as is? If not it would be better to leave it. If so then we should adjust it if the GEO_TYPE is PDB, otherwise this is now broken for other recipes